src/main/java/com/gmail/goosius/siegewar/events/SiegeWarStartEvent.java

```diff
@@ -0,0 +1,48 @@
+package com.gmail.goosius.siegewar.events;
+
+import com.palmergames.bukkit.towny.object.Nation;
+import com.palmergames.bukkit.towny.object.Town;
+import org.bukkit.block.Block;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+public class SiegeWarStartEvent extends Event {
+
+    private static final HandlerList handlers = new HandlerList();
+    private final Nation attackerNation;
+    private final Town attackerTown;
+    private final Town defenderTown;
+    private final Block flag;
+
+    public SiegeWarStartEvent(Block flag, Nation attackerNation, Town attackerTown, Town defenderTown){
```

Comment:
This should actually probably just be
```java
public SiegeWarStartEvent(Siege siege, Town town, Block flag) {
    this.siege = siege;
    this.attackerNation = siege.getAttackingNation();
    this.attackerTown = town;
    this.defenderTown = siege.getDefendingTown();
    this.flag = flag;
}
```
So that anyone using the API will have the actual Siege object handy.